src/Sarif.FunctionalTests/SpecExamples/RuleMetadata.sarif

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0"
+      },
+      "rules": {
+        "C2001": {
+          "id": "C2001",
+          "fullDescription": "A variable was used without being initialized. This can result in runtime errors such as null reference exceptions"
```

Comment:
we should build out this example to include more rules metadata, especially since this file is small